packages/rum-core/src/domain/rumEventsCollection/view/trackViews.spec.ts

```diff
@@ -271,6 +256,27 @@ describe('initial view', () => {
         expect(initialView.last.loadingTime).toBe(FAKE_NAVIGATION_ENTRY.loadEventEnd)
       })
     })
+
+    it('should not update timings long after the view ended', () => {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Nice!